Sephora-M/assignment1/AnagramsTest.java

```diff
@@ -0,0 +1,18 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class AnagramsTest {
+
+	@Test
+	void test() {
+		assertEquals(Anagrams.checkAnagrams("abcddd ", "dcba", false), false);
```

Comment:
You can also use assertTrue and assertFalse here instead of assertEquals.

Reply:
Good point, thanks - I'll keep that in mind.